src/sentry/static/sentry/app/views/settings/organizationRelays/modals/handleXhrErrorResponse.tsx

```diff
@@ -0,0 +1,71 @@
+import {t} from 'app/locale';
+
+type Error = {
+  type:
+    | 'unknown'
+    | 'bad-structure'
+    | 'missing-name'
+    | 'empty-name'
+    | 'missing-key'
+    | 'invalid-key';
+  message: string;
+};
```

Comment:
now there is also a "duplicate key" error

Reply:
I'll add it 😉